config.go

```diff
@@ -1,11 +1,12 @@
 package apigen
 
 type Config struct {
-	LocalRepo string
-	Org       string
-	Repo      string
-	Tag       string
-	TargetDir string
+	LocalRepo      string
+	Org            string
+	Repo           string
+	Tag            string
+	TargetDir      string
+	AdditionalDirs []string
```

Comment:
What additional dirs you had in mind?

Even what it got copied in my tests (current wip branch of cluster-api-gen) is too much, as the idea is to get only the minimum number of packages required to get `api`.

Reply:
e.g.
```
goclone --org kubernetes-sigs --repo cluster-api-provider-aws --tag v1.0.0 --target-dir ./out --additional-dir exp/api`,
```